compiler/modules/nback.llvm/llvm.bal

```diff
@@ -566,27 +566,40 @@ public class Builder {
     }
 
     // Corresponds to LLVMBuildGEP
-    public function getElementPtr(PointerValue ptr, Value[] indices,"inbounds"? inbounds=(), string? name = ()) returns PointerValue {
-        if indices.length() > 1 {
-            panic error ("More than one index not supported");
-        }
+    public function getElementPtr(PointerValue ptr, Value[] indices, "inbounds"? inbounds = (), string? name = ()) returns PointerValue {
         BasicBlock bb = self.bb();
         string reg = bb.func.genReg();
         string[] insWords = [];
-        insWords.push(reg);
-        insWords.push("=");
-        insWords.push("getelementptr");
-        insWords.push(typeToString(ptr.ty.pointsTo));
-        insWords.push(",");
-        insWords.push(typeToString(ptr.ty));
-        insWords.push(ptr.operand);
+        insWords.push(reg, "=", "getelementptr");
```

Comment:
`ins` is not the right abbreviation for `instruction` (`ins` is short for insert). Use `insnWords` or (better, since it's a local variable) just `words`.

Reply:
Fixed